docker-compose.dev.yml

```diff
@@ -116,6 +119,7 @@ services:
     profiles:
```

Comment:
We'll need to set an anchor for `bh-api` to use it in a separate `bh-api-sso` definition:

``` yaml
    bh-api: &bh-api
```